static/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
```

Comment:
remove this file?

Reply:
I have enabled the web UI as well to debug the collected metrics.